js/create-routine.js

```diff
@@ -0,0 +1,125 @@
+var todosDispositivos = ["Ar Condicionado", "Câmera", "Lâmpada",  "Rádio", "TV"];
+var dispositivosAindaDisponiveis = todosDispositivos.slice();
+var dispositivosSelecionados = [];
+var existeBotaoExcluir = false;
+var numeroCamposDispositivos = 0;
+
+var abreNovoCampoAcao = function(){
+    var camposAcao = document.getElementById("campos_de_acao");
+    var campoVisivel = camposAcao.lastElementChild; 
+        while (campoVisivel) {
+            camposAcao.removeChild(campoVisivel);
+            campoVisivel = camposAcao.lastElementChild;
+        }
+    var acionamento = document.getElementById('tipo-acionamento').value;    
+        if (acionamento == 'horario'){
+            document.getElementById('campos_de_acao').innerHTML = `<div class="form-group col-md-10 offset-md-1 mb-4">
+            <div class="row">
+                <div class="form-group col-md-8 mb-2">
+                    <label for="certa_distancia">⏰ Informe o horário do primeiro disparo:</label>
+                    <input type="datetime-local" class="form-control" id="certa_distancia" placeholder="">
+                </div>
+                <div class="form-group col-md-4">
+                    <label for="frequencia_acionamento">🔁 Frequência</label>
+                    <select class="form-control" id="frequencia_acionamento">
+                        <option value="">Apenas uma vez</option>     
+                        <option value="">Diariamente</option>
+                        <option value="">Apenas dias úteis</option>
+                        <option value="">Próximos 3 dias</option>
```

Comment:
Eu não to conseguindo abrir essas opções no frontend, elas não aparecem pra mim ;-;